### DISCUSSION TITLE
👍 Solved : Omada V3 script available?

### DISCUSSION THREAD

**Discussion by daad919** [tteck/Proxmox#1413] [Help Needed] [answered]
I have a couple of older omada AP , that i want to setup in a separate proxmox LXC. I've tried to do this manualy but i'm at best a linux noob. Any chance that there is a script available for older versions of the omada controller. This way we don't need to bin older equipement that is still dooing fine.

Thanks in advance.

**Comment by tteck** [comment, marked as answer]
Once the Omada v5 LXC has been created, execute the following commands one by one.

Version 3
```
apt-get remove omadac
wget -qL https://static.tp-link.com/upload/software/2022/202208/20220818/Omada_Controller_v3.2.17_Linux_x64.deb  
dpkg -i Omada_Controller_v3.2.17_Linux_x64.deb
rm Omada_Controller_v3.2.17_Linux_x64.deb
```
Version 4
```
apt-get remove omadac
wget -qL https://static.tp-link.com/upload/software/2021/202110/20211011/Omada_SDN_Controller_v4.4.6_Linux_x64.deb
dpkg -i Omada_SDN_Controller_v4.4.6_Linux_x64.deb
rm Omada_SDN_Controller_v4.4.6_Linux_x64.deb
```

**Reply by daad919** [reply]
This did the trick! 

Many thanks!

**Reply by goncalodumas** [reply]
Thank you for this! It allowed me to use older hardware for a secondary wifi network.
